packages/block-library/src/video/edit.js

```diff
@@ -232,8 +232,7 @@ function VideoEdit( {
 								}
 								render={ ( { open } ) => (
 									<Button
-										// TODO: Switch to `true` (40px size) if possible
-										__next40pxDefaultSize={ false }
+										__next40pxDefaultSize
```

Comment:
This can be seen while you add `core/video`, and check on inspector control while you need to add a poster image.

| Before | After     |
| :-------- | :------- |
| <img width="330" alt="Video-edit-before" src="https://github.com/user-attachments/assets/01f54800-f9ea-47dd-8255-5b5938c7b0a7"> | <img width="330" alt="video-edit-after" src="https://github.com/user-attachments/assets/cdcd891a-9d5d-421c-b563-f1d54f0e0454"> |